Lib/netrc.py

```diff
@@ -187,6 +187,3 @@ def __repr__(self):
                 rep += line
             rep += "\n"
         return rep
```

Comment:
Revert deleting the CLI entry point logic:
```suggestion
        return rep

if __name__ == '__main__':
    print(netrc())
```